chore: Update Usual informations

Magicking opened the pull request:
This pull request includes updates to the `data3.ts` file in the `defi/src/protocols` directory, specifically for the protocol information of "Usual Money". The most significant changes involve updating the protocol's name, address, symbol, description, audit status, gecko ID, cmcId, supported chains, and oracles.

Key updates to protocol information:

* Updated the protocol name from "Usual Money" to "Usual".
* Added the address `0xC4441c2BE5d8fA8126822B9929CA0b81Ea0DE38E` and updated the symbol to "USUAL".
* Enhanced the description to provide more detailed information about the protocol's decentralized banking framework and its stablecoin, USD0.
* Changed the audit status from "0" to "2" and added `gecko_id` "usual" and `cmcId` "33979".
* Expanded the supported chains to include "Arbitrum" alongside "Ethereum" and listed new oracles "Chainlink" and "Pyth".

realdealshaman commented:
I've added oracle info like this:
    oraclesByChain: {
      ethereum: ["Chainlink"], // https://tech.usual.money/smart-contracts/utility-contracts/chainlink-oracles
      arbitrum: ["Pyth"] // https://tech.usual.money/smart-contracts/utility-contracts/pyth-oracles
    },

Magicking commented:
Thanks for your addition. I updated the PR following your changes; it seems that the protocol is now named "usual" instead of "usual-money" according to the icon filename, shall all mentions be updated as well (adaptors, ref, and such) ?
It feels like some elements we updated previously are no longer displayed on the Defilama website

realdealshaman commented:
Hey @Magicking icons seem to be good now, if you still see the old one, you'll just need to clear your local cache. Regarding the chains reference, to add Arbitrum, it first has to be added to the adapter that calculates the TVL. Other than that, the PR looks ready to merge
